FinalProject/SessionService.cpp

```diff
@@ -270,12 +273,20 @@ DWORD SessionService::readFile(LPVOID input) {
 
 	//It should equal the buffer size - 1 to give room for null character
 	while (ReadFile(hFile, ReadBuffer, 1016, &dwBytesRead, NULL)) {
-		if (dwBytesRead == 0) {
+		if (dwBytesRead == 0)
```

Comment:
Deleted opening brace?

Reply:
doesn't show here but there's a tab in front of break, so that should work. I'll add it back

Reply:
Pushed changes.